my-webxr-app/src/components/DataPointMenu.tsx

```diff
@@ -32,7 +33,11 @@ export default function DataPointMenu(
           {`${selectedDataPoint?.color ?? '-'}\n`}
           x, y, z:
           {' '}
-          {`${selectedDataPoint?.meshProps?.position ?? '-'}\n`}
+          {`${selectedDataPoint?.meshProps?.position
+            ? `${(selectedDataPoint.meshProps.position as THREE.Vector3).x.toFixed(2)}, 
+            ${(selectedDataPoint.meshProps.position as THREE.Vector3).y.toFixed(2)}, 
+            ${(selectedDataPoint.meshProps.position as THREE.Vector3).z.toFixed(2)}`
```

Comment:
Can you try this on line 39: `${(selectedDataPoint.meshProps.position as THREE.Vector3).z.toFixed(2)}`.trim()

Reply:
still no luck with that

Reply:
Hopefully Zander's changes fixed the problem!